lib/fake_api_response_wrapper.rb

```diff
@@ -1,4 +1,12 @@
+require "ostruct"
```

Comment:
[praise] TIL!  I have never had occasion to use this before but it's pretty cool